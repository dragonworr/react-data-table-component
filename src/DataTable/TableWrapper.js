import styled from 'styled-components';

const TableWrapper = styled.div`
  position: relative;
  height: 100%;
`;

export default TableWrapper;
