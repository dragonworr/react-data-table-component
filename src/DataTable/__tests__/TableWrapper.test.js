import 'jest-styled-components';

import React from 'react';
import { shallow } from 'enzyme';

import TableWrapper from '../TableWrapper';

test('component <TableWrapper /> should render correctly', () => {
  const wrapper = shallow(<TableWrapper />);
  expect(wrapper).toMatchSnapshot();
});
