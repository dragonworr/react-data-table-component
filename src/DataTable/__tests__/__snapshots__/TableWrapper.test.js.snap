// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <TableWrapper /> should render correctly 1`] = `
.c0 {
  position: relative;
  height: 100%;
}

<div
  className="c0"
/>
`;
